types/undertaker/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "async-done": "^1.3.2",
+        "rxjs": "^6.6.3"
```

Comment:
Why does this depend on `rxjs`? That's a fairly large dependency and will slow down testing, and as far as I can tell it isn't even needed since `async-done` have its own minimal definition for `Observable` that it depends on: https://github.com/gulpjs/async-done/blob/master/index.d.ts#L80

Reply:
Yes, it's for the test. In my opinion, testing should be as independent as possible from any known definitions (i.e. do black-box testing), but since it slows down testing ...

Reply:
As written it would also have caused everyone who installed `@types/undertaker` to transitively get a copy of `rxjs`.

Reply:
Yes that's true, and on Windows with a default cluster size of 4 KB, it would take up about 7.05 MB on disk (that is the size of the `node_modules` folder with only `rxjs` 6.6.3 installed).

Reply:
I think it's more accurate to say "... it would take up a maximum of 7.05 MB ..." but I'm not that sure about that.